README.md

```diff
@@ -41,8 +41,12 @@ We appreciate sample model contributions; they help ensure a consistent glTF eco
 
 To contribute a model, open a pull request with:
 * A new subdirectory containing
-   * The model in as many glTF variations as possible (using the same directory structure as the others ([example](1.0/Box))).  Tools for converting to glTF are [here](https://github.com/KhronosGroup/glTF#converters).
-   * A README.md with any usage restrictions.
+   * The model in as many glTF variations as possible (using the same directory structure as the others ([example](2.0/Box))).  Tools for converting to glTF are [here](https://github.com/KhronosGroup/glTF#converters).
```

Comment:
Instead of "possible" can we say "reasonable" here?  Large models omit variations to keep the size of the repo down.

Reply:
I fixed it, but it is a bit subtle and subjective, of course: There are small models that are only available as `glTF` as well. But in any case, that's the baseline, and in doubt, the available tools usually make it easy to create the embedded/binary variants of it.